rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -164,28 +151,32 @@ def apply_rough_alignment(render,
 class ApplyRoughAlignmentTransform(RenderModule):
     default_schema = ApplyRoughAlignmentTransformParameters
     default_output_schema = ApplyRoughAlignmentOutputParameters
+    
     def run(self):
-        if self.args['minZ'] > self.args['maxZ']:
-            raise RenderModuleException("First section z cannot be greater or equal to last section z")
+        #if self.args['minZ'] > self.args['maxZ']:
```

Comment:
I think we should delete the old code in the comments.. we have it in version control so we can get it back easily.